packages/ui-components/styles/index.css

```diff
@@ -13,15 +13,16 @@
 @import './base.css' layer(utilities);
 @import './markdown.css' layer(utilities);
 
-/*
```

Comment:
Any reason why we are removing this comment?

Reply:
Yes, we've moved the comment to individual "See" comments above the relevant CSS